src/frontend-scripts/components/section-main/Settings.jsx

```diff
@@ -1212,6 +1213,53 @@ class Settings extends React.Component {
u2F5rWDS5ii+LELZZkNL6uh2p1DOkLTyJ75jfNgfHp1eFoseugWT0wYvvAmQkmQnW3vvGtl3sP255A5uMnkXnjl82DuQiHyiC4GYcW3wMhA5DBXghTqZdaKoTA/LVPK03n06/LtjMYv/AEsqcfnhuYi5dQi0dqREGZBhFYBxHZAs4F9MQ+mPo2ECKWu3Bc9nxq8VouDmF86DFMDTw3PzCfXpVQUG6A6GFQwhDqO45o+x2gVIUWWqd5cGDbaeTTryL7zhOoZM5dJZjPjD2859uIdx0FZUqdWqN5XgW2M4FiYRCTH/wVxaGX/LVUU7NSs0MM9j+AROo4WAuAcd0inEoWdukTlErvojg5iOKHp+DkP5pWiIWqdgj2fwuJ1LGWAFMufYCp0ddQHnsTleKHsEvZGVAWVGLCN3wH8c67W0KVCvm3kRl8BIUP/v6p6nxWMxo6eI1ViYBIC6FdJxDvONIyYNIDv0Ph/ZdnHd78wajgN9f9uK+W3A8zuRfR2Jf82O5yb4Wps0gP/B6F915aKDC1EKdKj1BhQSb6/biv1fMVaCICKXvqaZLljaZcHkT6/B+QG/jLwoBRzp0fCI9ugGZ1wkx+EVw5ekY3GFrHf6m6I8gM/gkTZx+ZfwTvUqRMBGB27Edw6zFILQIpVESv9ZoCM/H+U8j+53E/T3WlNicb44PpPIDwlmNgwgTXki0lKRcBqMxCbuwfyA4+i2rm3BXhzA2M1QGj62bo7dughzZA07a2JBgVg6mUP0E+/U8Ux15HeeLd5kKblFt+6vViI1oQzGyDDG2A0XkzhIhCM7vAabRlV9MqeufYKrR5DlPpMyh8+OrsoU0VrWNaAMLshBAJUKL5KVVihMHNDgi9DdLoASW1aXoxMo5La7Fcf61UyP8X2eGXUKyvrGd4vgqKue4Q9MBGSD0BrsDAUIG/GgSVqya8lsRHK2cLZqJX46uULyA9/DSm3nnmU3vj2xiVjzY3q9zRYehGL6hfmaAAXCzb8Bku7ctcwrs59iRy6VeQPf8MyqOnp1fXWrwfka13wQrvAeORFSURzfD1PBuV0jDSw08h5/s29WC4te1uxNcfhdCSIGThKxWaebhr3cdxc8iOncTYqz+bBhPYdS8S3feAc1XIszqbyi/ls6fw8cs/mgYT3HUfEt3HwVYzGHcKk5kXMXLqJ9NgwrvvR3vq6KoG4ziTmBh9HunXfwHPKdcSbuH++9HetbrBuG4euczfkP73H1HNDtTABK47isSG+8BFeHUaGLXG9myUS0MYH3wUU+efq4HRkweQ3PkDCL1jRfsrjd66KlkbHX4Ek2ceroGRgR507P8hdGs7CFovYdZowM1+r8CMX3gME6d/VQNDRRDt+7+HUOQ2UGo1e50V10+BGbvwKLKnfz1d7RDd+wCinUfAeWTFDbjZAV2mSuqHgb57EO/5GoSMNXudFdevBuZRTJ65RGL01CEk+h6CZvSsgHDC/N6Z6xYwkX4BY6/8dFqVmBZFfP/3EQgfACXLPwUyv6F//q88VFEovo3hkw9dWlFF0H7TdxGK3wnOVqedqYE5i+GTD84sNYvsPoFo110QIr4YL2TZX1Ml+7OZFzF66sczwQT7jiG27jikphy91dccJ4/M6LOYOP3bmWCMdbehfesJ6NZmf5PVamuuW0Yh9wbS5x6fCYZbKbTtuhfB2OEVWLbazGt2YVczGBv+80wwKtAd2nEc0e4jkGJ1qpOSmszYM5fX+Rrrb0e056swQztaoqSjGTmYSx8FJj329OVgRGg9zORBWB2HoVkpMBZsidKOuQz+8/q6bgkjH33GxqgfqPyRCKSgxXdDBruhBzdDKG9YlX6oUlZqrmjPWEnM2MhTVy6Z92t5tTCotKBFt0GENoExCaGykYFeP19NaG3XLKEaKLSFemHX/Dqq5jef/1fjvQRUj4BqEX+rMA+mIGO7QIkAVztjhQGudUBo3f6uWSVVfqaSyLpUtWKSzoPa9DWnagciA2Bmor51mINwDcJaBxHtAyUUjEswGfRVj1KVzWSglAMqq6nSuy2S2lX7meYE5jI5V3urRRA00OHvu6dc+NXiWnwfqIyCUVGro5ExMNHuz3I1yeJ1UMvVifSuEswsFoGHe0C4Wd+pr4HrCYhgL5iRAGPClyoFSxUO1KRq+ZWpXZ3ENGkqCdPBzHZQte+A8poUmSno8RshzXbo+ibfLi2ntiRgrjRgZa9kYies1AEEIv3QtS3LicviqFKzIyTqCITkDQj03Iq2tjub/dmS9LtmEuM7kzJQA7PuFrS13bEkA272JtcMjNotJzv3wuo+gEBoNzS5odlnXpJ+SwpGedTMjIMFOiADnTCSh6Dr3dD03mXn4ywJGBbo9B1DroWhR7dDhrdDqClb7wSnkWVZZr9oYFRJrDpWhekhaLHroYW3QYgQpNFZ91/qRzQtwlkzC6FrCw6GRzaCch3STEC27YQwk5BmF4TW4ZfH+kWP/nlVy7stABi1LAj4ywImNJjJW8FlFMKIQhrrwXgYhEp/4dlKlZ/zBqPOnWLBbt/tl8H1kG27IDQLWmgHOAvWz6XSWzZ2MycwRFi+aw9CIYwYjM4v+KeTacEuSGMTqH9Ilzq4a/mrSiNFbgqM8jnUJnRhJSHbd4JRE9KIQQ/3gyiZEEb9KINWjL9cGdGsYJRd8DdXSBMi1FuL5Jkd0EJbwWgQjOsrOsVyGZiLQLgRg4xs8eO+WnAjpNXrbx32p+EVFMqcTaVmgFFhTG7EIaPbIEM90EPbIc1uMAVD2Y4WicA1sh/NfO+D8aVEmtA69/kbtIxoP6SRBOcqftI6RxI0M+Bm+5B1vX2eqo2RsT5Y3Yehmz2QIrkso2rNDmoh+pFNN3/dU7vVrI5boFvdYCzcsr7HQgC5eA2y88jPPdO6Doa5acZ2v4W8SSteixz85klPM1TMdXVux5l1Vrr9wSGvldYwSyV95PYH32vuuNKleqJlcp//A00UAhSS3ax0AAAAAElFTkSuQmCC';
```

Comment:
You should never hardcode a base64 string like this. You should make a new server-side socket event called `resetCardback` (with proper authorisation) that effectively does the same as the mod tools event. 

If the default cardback is ever changed, this will break. This also doesn't *reset* the cardback, it *sets* it to a specific image.

Reply:
I see, I will be busy for few more months, but during summer I should be able to find some time to change it.